src/pages/community.tsx

```diff
@@ -150,7 +150,7 @@ const Community = () => {
                   joined our Community Slack
                 </a>
               </p>
-              <div className={paCss.custom_box}>
+              <div className={`${paCss.custom_box} ${paCss.text_center}`}>
```

Comment:
```suggestion
              <div className={clsx(paCss.custom_box, paCss.text_center)}>
```

use `clsx` utility when defining multiple classnames. It is already
imported in this page.

I'm aware the usage is not consistent throghout this page, but we'd like
to use `clsx` instead of string template. Less brackets and easier to
automate refactoring.